### PR TITLE
Update near-duplicate sets

### PR DESCRIPTION
**Pull request by elisno**
After this PR:
- Examples that are not flagged with `is_near_duplicate_issue = True` will have an empty near-duplicate set.
- The near-duplicates are now symmetric.

### CODE REVIEW

File: cleanlab/datalab/issue_manager/duplicate.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def _neighbors_within_radius(knn_graph: csr_matrix, radius: float):
         N = knn_graph.shape[0]
         distances = knn_graph.data.reshape(N, -1)
         # Create a mask for the threshold
-        mask = distances < radius
```
Comment (jwmueller) [inline cleanlab/datalab/issue_manager/duplicate.py:128]: can you confirm the set of examples flagged as near duplicates has not changed, i.e. this PR only updates the sets of near_duplicates in the info object?

File: tests/datalab/issue_manager/test_duplicate.py [hunk 2]
```diff
@@ -80,3 +104,49 @@ def test_report(self, issue_manager, embeddings):
...
+        assert all(
+            len(near_duplicate_set) == 0
+            for i, near_duplicate_set in enumerate(near_duplicate_sets)
+            if not issues[i]
+        ), "Non-issue examples should not have near duplicate sets"
+        assert all(
+            len(near_duplicate_set) > 0
+            for i, near_duplicate_set in enumerate(near_duplicate_sets)
+            if issues[i]
+        ), "Issue examples should have near duplicate sets"
```
Comment (jwmueller) [inline tests/datalab/issue_manager/test_duplicate.py:152]: think you could easily combine these into one test for runtime efficiency, but not important if it's not easy to do

**Review by jwmueller — APPROVED** [review]
Note that `black` format is not passing in CI, so update that first.

Otherwise LGTM, feel free to merge once you confirm the questions I had!

File: tests/datalab/issue_manager/test_duplicate.py (hunk 2 above)
Comment (elisno) [reply tests/datalab/issue_manager/test_duplicate.py]: Combined tests as they share the same setup code.

File: cleanlab/datalab/issue_manager/duplicate.py (hunk 1 above)
Comment (elisno) [reply cleanlab/datalab/issue_manager/duplicate.py:128]: The `issues` dataframe that flags near duplicates is set only once in the `find_issues` method. This occurs before the call to the `_neighbors_within_radius` which computes the sets of near-duplicates (which ends up in the info object).

I can confirm: The flagged examples have **not changed**.

**Pull request merged**